### DISCUSSION TITLE
Should attachments appear in co-citations?

### DISCUSSION THREAD

**Discussion by HEmile** [SkepticMystic/graph-analysis#21] [Ideas]
Currently, co-citations can only be other notes. Should linked attachments also be possible co-citations?

**Comment by SkepticMystic** [comment]
I think this should be implement with the option to disable.
On the code side, we would need to:
1. `initGraph` with any file extension included (currently we filter only for `path.endsWith('.md')`
2. Check that, when the active note is an image, for example, the components don't require an active md note to work

**Comment by SkepticMystic** [comment]
For anyone interested, this is now possible in 0.7.0!
Give it a try, and let us know if you run into any bugs :)
